book/src/lints.md

```diff
@@ -101,5 +101,18 @@ The `clippy::cargo` group gives you suggestions on how to improve your
 your crate and are not sure if you have all useful information in your
 `Cargo.toml`.
 
+## Nursery
+
+The `clippy::nursery` group contains lints which are buggy or need more work. It is **not** 
+recommended to enable the whole group, but rather cherry-pick lints that are useful for your 
+code base and your use case, take to account lack of maturity such lints. 
```

Comment:
The last sentence is a bit weird.
```suggestion
code base and your use case.
```
I would just leave it out, as this is already mentioned in the first sentence.